packages/ember-views/tests/views/view_test.js

```diff
@@ -86,53 +79,3 @@ test('should re-render if the context is changed', function() {
 
   equal(jQuery('#qunit-fixture #template-context-test').text(), 'bang baz', 're-renders the view with the updated context');
 });
-
-test('propagates dependent-key invalidated sets upstream', function() {
```

Comment:
This now tested [here](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L2184-L2318)